packages/clay-form/src/InputWithAutocomplete.tsx

```diff
@@ -33,7 +35,7 @@ interface IProps extends React.InputHTMLAttributes<HTMLInputElement> {
 	/**
 	 * Callback for when input value changes
 	 */
-	onItemSelect: (val: string | Object) => string | Object;
+	onItemSelect?: (val: string | Object) => void;
```

Comment:
Would we ever actually have a case where a user wouldn't pass an `onItemSelect`?

Reply:
Oh sorry, actually I had done this to try using Autocomplete instead of the search but had trouble adjusting the markup. I will remove this, I do not remember why I had to add this as optional. But I think we can keep the `void` return here still.